src/axolotl/utils/collators/mm_pretrain.py

```
@@ -0,0 +1,306 @@
+            offender_idx: Optional[int] = None
+            retry_ok = True
+            retry_kwargs: dict[str, Any] = {
+                "return_tensors": self.return_tensors,
+                "padding": self.padding,
+            }
+            if self.pad_to_multiple_of is not None:
+                retry_kwargs["pad_to_multiple_of"] = self.pad_to_multiple_of
+            for i, (t, imgs) in enumerate(zip(texts, images, strict=True)):
+                try:
+                    self.processor(text=[t], images=[imgs], **retry_kwargs)
+                except Exception as retry_exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)